src/stream/src/executor/dispatch.rs

```diff
@@ -270,14 +305,17 @@ impl DispatchExecutorInner {
                     }
                 }
             }
-
+            Mutation::Combined(v) => match &v[..] {
+                [Mutation::Add { .. }, Mutation::Update { .. }] => {
```

Comment:
Why not just change the mutation to a list of mutations in proto?

```diff
 message Barrier {
-  BarrierMutation mutation = 3;
+  repeated BarrierMutation mutations = 3;
```

Reply:
Any updates?

Reply:
Any updates here? 👀

Reply:
However, is it general enough for all executors to correctly handle it?